app/lib/i18next/i18next.d.ts

```diff
@@ -0,0 +1,7 @@
+import 'i18next'
+
+declare module 'i18next' {
+  interface CustomTypeOptions {
+    returnNull: false
+  }
+}
```

Comment:
As discussed - implement an explicit check.

Reply:
Fixed in [f1cf05e](https://github.com/secvisogram/secvisogram/pull/433/commits/f1cf05e13996aa5ec04656c6a448ef43e249c4da).